src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -202,23 +202,23 @@ and a guide for using macros can be found at
 If the structural images included in the dataset were defaced (to protect
 identity of participants) one MAY provide the binary mask that was used to
 remove facial features in the form of `_defacemask` files.
-In such cases,  the OPTIONAL [`mod-<label>`](../99-appendices/09-entities.md#mod)
+In such cases, the OPTIONAL [`mod-<label>`](../appendices/entities.md#mod)
 entity corresponds to modality suffix,
 such as `T1w` or `inplaneT1`, referenced by the defacemask image.
 For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 
 If several scans with the same acquisition parameters are acquired in the same session,
```

Comment:
am sure you thought that but maybe worth thinking of merge order for this PR, given that some of those lines are nuked by PR https://github.com/bids-standard/bids-specification/pull/1265

Reply:
Yeah... I'm not sure what the best order to merge them in is though. I think I'll be dealing with some gnarly merge conflicts no matter what.